source/js/main.js

```diff
@@ -146,6 +146,56 @@ let main = {
       MozfestCarousels.init();
       MozfestHeroCarousels.init();
     }
+
+    if (document.querySelector(`.donate-banner`)) {
+      document
+        .querySelector(`.donate-banner .banner-close`)
+        .addEventListener(`click`, (e) => {
+          document.querySelector(`.donate-banner`).style.position = "absolute";
+          document.querySelector(`.donate-banner`).style.top = "0px";
+          e.preventDefault();
+
+          const animationOptions = {
+            // timing options
+            duration: 300,
+            fill: "forwards",
+            easing: "ease-in",
+          };
+
+          document.querySelector(`.wrapper`).animate(
+            [
+              // keyframes
+              {
+                marginTop:
+                  document.querySelector(`.donate-banner`).clientHeight + `px`,
+              },
+              {
+                marginTop: 0,
+              },
+            ],
+            animationOptions
+          );
+
+          const closeAnimation = document
+            .querySelector(`.donate-banner`)
+            .animate(
+              [
+                // keyframes
+                {
+                  transform: `translateY(0)`,
+                },
+                {
+                  transform: `translateY(-100%)`,
+                },
+              ],
+              animationOptions
+            );
+
+          closeAnimation.onfinish = () => {
+            document.querySelector(`.donate-banner`).style.display = "none";
+          };
```

Comment:
Let's make sure that this banner stays hidden until "tomorrow" with a bit of `localStorage` work:

```suggestion
          closeAnimation.onfinish = () => {
            document.querySelector(`.donate-banner`).style.display = "none";
            localStorage.setItem(DISMISS_KEY, getDay());
          };
```

with a
```
const DISMISS_KEY = `donate-banner-dismiss-day`;
```
up top somewhere, as well as a simple

```
function getDay() {
    return (new Date()).getDay();
}
```